src/StickerService.ts

```diff
@@ -50,6 +53,48 @@ function buildWallpaperCss({
     background-size: cover !important;
   }
 
+  /* Output panel */
+  .monaco-workbench .part.panel > .content .monaco-editor .monaco-editor-background,
+  .margin-view-overlays, 
+  .margin, 
+  /* Welcome screen */
```

Comment:
I am a bit curious as to what this does. If I omit it, and just include 
```css
.editor-container
  {
    background-image: url('file:///home/alex/.config/Code/User/globalStorage/unthrottled.doki-theme/wallpapers/ryuko.png') !important;
    background-position: right !important;
    background-attachment: fixed !important;
    background-repeat: no-repeat !important;
    background-size: cover !important;
  }
```
The welcome looks like this:

![image](https://user-images.githubusercontent.com/15972415/114386229-912bed80-9b56-11eb-8928-20c40519a532.png)